docs/en/api/POSIX-API.md

```diff
@@ -462,3 +462,204 @@ When encountering the out of direct memory issue, add the following JVM opts to
+For example, one tensorflow application do some read operations against the Fuse mount point and do training.
+To make sure whether the interaction with Alluxio POSIX API is the performance bottleneck, we do the following testing
+- Compare the data read throughput between original application and the one that removed the training logic.
+- Compare the data read throughput between Alluxio POSIX API and other data solutions.
+- Ruled out the other possible factors. For example, compare the data read throughput between current resources and the enlarged one.
+
+### Fuse
+
+The Fuse components, especially the libfuse and FUSE kernel code, are known to have performance overhead.
+Instead of Alluxio Fuse which interacts with Alluxio, [Stack Fuse](https://github.com/Alluxio/alluxio/blob/ea36bb385d24769e079248015c8e490b6e46e6ed/integration/fuse/src/main/java/alluxio/fuse/StackFS.java)
+interacts with local filesystem directly.
+
+For example, Alluxio Fuse read go through `Fuse kernel code -> libfuse -> JNIFuse -> AlluxioFuse -> Alluxio client -> Alluxio worker -> read the data stored on Alluxio Worker storage (local filesystem)`.
+Stack FUSE read go through `Fuse kernel code -> libfuse -> JNIFuse -> StackFS -> read the data stored on local filesystem`.
+Local filesystem read go through nothing but directly read from local filesystem.
+As long as the medium type and medium condition are similar among the three read type testing, they are comparable.
+
+By comparing the throughput between local filesystem operations and Stack Fuse operations, we can understand the overhead introduced by Fuse components (Fuse kernel, libfuse, and JNIFuse).
+By comparing the throughput between Stack Fuse operations and Alluxio Fuse operations, we can understand the overhead introduced by Alluxio itself and what's the upper bound performance AlluxioFuse can achieve.
+
+According to the Alluxio whitepaper [design and implement Alluxio POSIX API](https://www.alluxio.io/resources/whitepapers/design-and-implementation-of-alluxio-posix-support/),
+JniFuse does not introduce much overhead.
+
+Launching Stack Fuse is similar to Alluxio Fuse.
+
+First, create the source directory with permission
+```console
+$ sudo mkdir sourceDir
+$ sudo chown $(whoami) sourceDir
+$ chmod 755 sourceDir
+```
+
+Second, create the mount directory as before
+```console
+$ sudo mkdir -p /mnt/people
+$ sudo chown $(whoami) /mnt/people
+$ chmod 755 /mnt/people
+```
+
+Mount the source directory to mount directory using `StackFS`
+```console
+# integration/fuse/bin/alluxio-fuse mount -s [mount_point] [source_dir]
+$ integration/fuse/bin/alluxio-fuse mount -s /mnt/people sourceDir
+```
+
+Check `StackFS` status, unmount `StackFS` exactly the same as Alluxio Fuse
+```console
+# Stat StackFS
+$ integration/fuse/bin/alluxio-fuse mount stat
+
+# Umount StackFS
+# integration/fuse/bin/alluxio-fuse umount [mount_point]
+$ integration/fuse/bin/alluxio-fuse umount /mnt/people
```

Comment:
It seems that the `integration/fuse/bin/alluxio-fuse stat` and `integration/fuse/bin/alluxio-fuse umount` doesn't work well for `StackFS`. Just tried myself, mount works, but can't see in the `integration/fuse/bin/alluxio-fuse stat`, and `integration/fuse/bin/alluxio-fuse umount /test/path` returned `/test/path not mounted` and I can still see if run `mount` command.

Reply:
Thanks for testing out, 
did you try in Linux or Mac? If not work, it's a bug that need to be fixed

Reply:
On Linux. Didn't try on Mac.